Shapes/Shape.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shapes
+{
+    public abstract class Shape
+    {
+        protected bool full;
```

Comment:
Booleans should look like a question
```suggestion
        protected bool isFull;
```